src/app/components/forms/form/form-container/terra-form-container.component.ts

```diff
@@ -125,6 +126,25 @@ export class TerraFormContainerComponent implements OnInit, OnChanges, ControlVa
             {
                 this.formFieldVisibility[field.key] = isNullOrUndefined(field.value.isVisible) || field.value.isVisible;
             }
+
+            if(!isNullOrUndefined(this.formGroup))
+            {
+                let control:AbstractControl = this.formGroup.get(field.key);
+                if(this.formFieldVisibility[field.key])
+                {
+                    if(control.disabled)
+                    {
+                        control.enable({onlySelf:true});
```

Comment:
kannste
` {onlySelf:true} `
nicht in eine Variable auslagern?

Reply:
Sollte nicht gemacht werden, weil dann alle Fields einer Control das gleiche Objekt bekommen. Das kann, muss nicht, zu unerwünschten Seiteneffekten führen (hier nicht, aber vielleicht in der Zukunft :) )